aiida_mlip/data/model.py

```diff
@@ -195,13 +155,13 @@ def local_file(
 
     @classmethod
     # pylint: disable=too-many-arguments
-    def download(
+    def from_url(
```

Comment:
technically probably from_uri is better than from_url

Reply:
Why do you think that? We want it from a `URL` `file:///home/...` is a `URL` too, just using a `file` protocol. Likewise `~/my_file` can be considered a `URL` with implicit `file` protocol